<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #E0F7FA;   /* Light cyan */
    color: #000000;              /* Text color */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/images/2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="input_btn">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>153</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/images/Input.png</normaloff>../resources/images/Input.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ANN_controller_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>118</y>
      <width>380</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/images/ANN-controller.png</normaloff>../resources/images/ANN-controller.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>375</width>
      <height>280</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="DC_motor_btn">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>198</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/images/DC-motor.png</normaloff>../resources/images/DC-motor.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Output_btn">
    <property name="geometry">
     <rect>
      <x>1135</x>
      <y>202</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/images/Output_Graph-removebg-preview.png</normaloff>../resources/images/Output_Graph-removebg-preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>145</width>
      <height>140</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="User_guide_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/images/user-guide-icon.png</normaloff>../resources/images/user-guide-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>91</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="sampling_time_edit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #E0F7FA;   /* Light cyan */
    color: #000000;              /* Text color */
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Restart_btn">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/images/restart_button.png</normaloff>../resources/images/restart_button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Run_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/images/start_button.jpg</normaloff>../resources/images/start_button.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
